microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -682,20 +679,57 @@ See also:
 * [Filter requests](../network/reference.md#filter-requests) in _Network features reference_.
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 120.
-* This checkbox is not present in Microsoft Edge Stable 118.
+* This checkbox is present in Microsoft Edge Canary 121.
+* This checkbox is present in Microsoft Edge Stable 119.
+
+
+<!-- ====================================================================== -->
+## Enable breadcrumbs in the Performance Panel
+<!-- no (?) link -->
+
+_new as of Microsoft Edge 121_
+
+Enables breadcrumbs in the **Performance** tool.
+<!-- todo: need sentence -->
```

Comment:
1. @captainbrosset @zoherghadyali - need info about the Experiments checkbox:
Enable breadcrumbs in the Performance Panel

Reply:
This checkbox adds a new feature to the Application tool:

<img width="569" alt="image" src="https://github.com/MicrosoftDocs/edge-developer/assets/1152698/ed09b7ac-3f42-43e7-a1bf-715fd02d85a8">

Now when you drag your mouse over the overview, to select a time range, you have the option of clicking on a magnifier icon in the middle of the range to zoom in on that range only, and ignore everything else. When you do, a new item is added to the breadcrumbs widget which I highlighted in the previous image. And you can then click on any item in the breadcrumbs widget to go back to a previous range.

Reply:
Here is what zooming in on a range looks like:

<img width="197" alt="image" src="https://github.com/MicrosoftDocs/edge-developer/assets/1152698/aadef8f7-c99f-46cb-a5dc-2b371f40fdb0">